src/components/inputCo.js

```diff
@@ -43,6 +20,9 @@ const useStyles = makeStyles(() => ({
     display: "flex",
     alignItems: "center",
   },
+  helperTextContainer: {
+    height: "20px",
```

Comment:
Could you use just the number instead? JSS default unit is pixel

Reply:
Done